client/packages/common/src/utils/functions/ApiUtils.tsx

```diff
@@ -4,9 +4,9 @@ export const setNullableInput = <NODE, KEY extends keyof NODE>(
   key: KEY,
   entity?: NODE | null | undefined
 ) => {
-  if (entity == null) return { value: undefined };
+  if (entity === null) return { value: null };
```

Comment:
Classic error caused by the less strict equality operator (`==`). This returns true for `undefined` as well, so properties that were not defined were replaced with `null`s.